server/handler/tables.go

```diff
@@ -2,18 +2,58 @@ package handler
 
 import (
 	"net/http"
-	"strings"
 
 	"github.com/src-d/gitbase-playground/server/serializer"
 	"github.com/src-d/gitbase-playground/server/service"
 )
 
-// Tables returns a function that calls /query with the SQL "SHOW TABLES"
+// Tables returns DB schema
```

Comment:
If `Tables()` returns a `DB schema`, why not call the function `Schema` or something more meaningful?

Reply:
PR is implemented according to the issue description. I think it's better to keep handler names and names of endpoints consistent.

Reply:
yep. No problem with renaming if we want to change the endpoint name too!

Reply:
The initial design doc defined `/tables` to return only the tables, now that we have added more complete functionality I'm ok with renaming everything to `/schema`. @smacker go ahead if you feel it fits in this PR, otherwise we can consider this rename in a future PR.